api/lib/controllers/meeting.js

```diff
@@ -265,68 +265,142 @@ module.exports = {
-                $lookup: {
-                  from: "meetings",
-                  localField: "meeting_id",
-                  foreignField: "_id",
-                  as: "meetings",
-                },
+      });
+
+    name &&
+      pipelineFilters.push({
+        $match: { name: { $regex: name, $options: "i" } },
+      });
+
+    const sliceStart = parseInt(skip);
+    const sliceEnd = parseInt(limit);
+
+    const pipeline = [
+      {
+        $match: { _id: ObjectID(subject_id) },
+      },
+      {
+        $lookup: {
+          from: "meetings",
+          localField: "_id",
+          foreignField: "owner_id",
+          pipeline: [...pipelineFilters, ...ownerLookup],
+          as: "owned_meetings",
+        },
+      },
+      {
+        $lookup: {
+          from: "participants",
+          pipeline: [
+            { $match: { email: subject_email } },
+            {
+              $lookup: {
+                from: "meetings",
+                localField: "meeting_id",
+                foreignField: "_id",
+                pipeline: [...pipelineFilters, ...ownerLookup],
+                as: "meetings",
               },
-              {
-                $project: {
-                  meeting: { $arrayElemAt: ["$meetings", 0] },
+            },
+            {
+              $project: {
+                meeting: {
+                  $arrayElemAt: ["$meetings", 0],
                 },
               },
-              {
-                $replaceRoot: {
-                  newRoot: "$meeting",
-                },
+            },
+            { $unwind: "$meeting" },
+            {
+              $replaceRoot: {
+                newRoot: "$meeting",
               },
-            ],
-            as: "participating_meetings",
+            },
+          ],
+          as: "participating_meetings",
+        },
+      },
+      {
+        $project: {
+          meetings: {
+            $concatArrays: ["$participating_meetings", "$owned_meetings"],
           },
         },
-        {
-          $project: {
-            meetings: {
-              $concatArrays: ["$participating_meetings", "$owned_meetings"],
+      },
+      {
+        $project: {
+          meetings: {
+            $sortArray: {
+              input: "$meetings",
+              sortBy: { date: -1 },
             },
           },
+          count: { $size: "$meetings" },
+          owners: "$meetings.owner",
+        },
+      },
+      {
+        $project: {
+          meetings: {
+            $slice: ["$meetings", sliceStart, sliceEnd],
+          },
+          count: 1,
+          owners: 1,
         },
-        { $unwind: "$meetings" },
-        { $replaceRoot: { newRoot: "$meetings" } },
-        { $sort: { date: -1 } },
-        { $skip: parseInt(skip) },
-        { $limit: parseInt(limit) },
-      ];
+      },
+    ];
 
-      const meetings = await User.aggregate(pipeline);
+    const [{ meetings, count, allOwners = owners }] = await User.aggregate(pipeline);
```

Comment:
```suggestion
    const [{ meetings, count, owners: allOwners }] = await User.aggregate(pipeline);
```
You use the `=` when unpacking when you want to have a default value.

Reply:
It doesnt work when I do it like that